lib/sinon/util/fake_timers.js

```diff
@@ -2,17 +2,34 @@
 
 var llx = require("lolex");
 
-exports.useFakeTimers = function () {
-    var now;
-    var methods = Array.prototype.slice.call(arguments);
+/**
+ *
+ * @param config {number|Date|Object} the unix epoch value to install with (default 0) or
+ */
+exports.useFakeTimers = function (args) {
```

Comment:
To me, this method is a bit difficult to read, because

* `for` loop for creating a new `Object` with the own properties from `args`. This can be done with [`Object.create`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/create) in a single statement
* nested conditionals make everything more difficult to read
* `typeof null` returns `"object"`, which would cause errors if you call the method with `null`.

I have made an attempt at addressing those issues, I think this is functionally equivalent

```javascript
exports.useFakeTimers = function (args) {
    if (typeof args === "string" || Array.isArray(args) && args.length > 1) {
        throw new TypeError("useFakeTimers expected 1 argument." +
            " see https://github.com/sinonjs/sinon");
    }

    var config;

    if (!Boolean(args)) {
        config = {
            now: 0
        };        
    }
    
    if (!config && typeof args === "number" || args instanceof Date) {
        config = {
            now: args
        };
    }

    if (!config && typeof args === "object") {
        config = Object.create(null, args);
    }

    if (!config) {
        throw new TypeError("useFakeTimers called with " + String(args) +
            " but expected epoch or config object. see https://github.com/sinonjs/sinon");
    }

    var clock = llx.install(config);
    clock.restore = clock.uninstall;
    return clock;
};
```

Admittedly, it has more lines, but I think it reads a lot easier with the reduced indentation (and no use of `else`).

What do you think?